src/modules/shareddrives/components/SharedDriveFolderBody.jsx

```diff
@@ -1,13 +1,71 @@
 import React from 'react'
+import { useDispatch } from 'react-redux'
+import { useNavigate, useLocation, useParams } from 'react-router-dom'
 
+import { useClient } from 'cozy-client'
+import { useVaultClient } from 'cozy-keys-lib'
+import { useSharingContext, useNativeFileSharing } from 'cozy-sharing'
+import { makeActions } from 'cozy-ui/transpiled/react/ActionsMenu/Actions'
+import { useAlert } from 'cozy-ui/transpiled/react/providers/Alert'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+import { useCurrentFolderId } from '@/hooks'
+import { useModalContext } from '@/lib/ModalContext'
+import { download, infos, versions, hr } from '@/modules/actions'
 import { FolderBody } from '@/modules/folder/components/FolderBody'
 
 const SharedDriveFolderBody = ({ folderId, queryResults }) => {
+  const navigate = useNavigate()
+  const { pathname } = useLocation()
+  const client = useClient()
+  const vaultClient = useVaultClient()
+  const currentFolderId = useCurrentFolderId()
+  const { driveId } = useParams()
+  const { t, lang } = useI18n()
+  const { pushModal, popModal } = useModalContext()
+  const { allLoaded, hasWriteAccess, refresh, isOwner, byDocId } =
+    useSharingContext()
+  const dispatch = useDispatch()
+  const { isMobile } = useBreakpoints()
+  const { showAlert } = useAlert()
+  const { isNativeFileSharingAvailable, shareFilesNative } =
+    useNativeFileSharing()
+  const canWriteToCurrentFolder = hasWriteAccess(currentFolderId)
+
+  const actionsOptions = {
+    client,
+    t,
+    lang,
+    vaultClient,
+    pushModal,
+    popModal,
+    refresh,
+    dispatch,
+    navigate,
+    pathname,
+    hasWriteAccess: canWriteToCurrentFolder,
+    canMove: true,
+    isPublic: false,
+    allLoaded,
+    showAlert,
+    isOwner,
+    byDocId,
+    isMobile,
+    isNativeFileSharingAvailable,
+    shareFilesNative,
+    sharedDriveId: driveId
+  }
```

Comment:
Are all these attributes necessary here? Or is it a copy past of another folder body component ?